.travis.yml

```diff
@@ -6,9 +6,9 @@ addons:
   chrome: stable
 
 node_js:
-  - "4"
   - "6"
   - "8"
+  - "10"
```

Comment:
Thanks for updating this!

Reply:
Well, I had to  in order to get a green light on the PR :)